contents/blog/aruba-hackathon.md

```diff
@@ -0,0 +1,185 @@
+
+Dubbed DataBeach because Frank and Harry started building the feature while sipping Piña Coladas by the beach, DataBeach is all about our long-term vision of [simplifying the modern data stack](/blog/modern-data-stack-sucks) for startups. Less time spent wrangling data = more time shipping products. 
+
+![databeach](../images/blog/aruba/data-beach.png)
+
+The MVP they built consists of custom tables that are created and queried through the PostHog UI and API. These tables provide a way to store and query data from sources like Stripe, Hubspot (see above), Intercom, and more, along with data from PostHog.
+
+There's more to build before PostHog is ready to be your data warehouse, but we're working on it. Keep an eye on our [public roadmap](/roadmap) for updates.
+
+## Leveling up PostHog support
+
+- **Team Arubug:** Tiina Turban, Simon Fisher, Paul D'Ambra, Cameron DeLeone, and Cory Watilo
+
+Anecdotally, our direct-to-an-engineer support is a big reason why people love us. But we've run into a few of inefficiencies as we've grown, such as:
+
+- Support requests that lack the context an engineer needs to fix it
+- Lack of trends and aggregated info about support requests
+- Difficulty managing multiple sources for support requests
+
+Enter team Arubug, who decided there had to be a better way.
+
+![bug report](../images/blog/aruba/bug-report.gif)
+
+The team started by [building a site app](/tutorials/build-site-app) for bug reports based on our existing [Feedback Widget](/apps/feedback-widget), which sends a `$bug_report` event to PostHog.
+
+// TODO: Screenshot of dashboard
+
+These reports feed into a dashboard that tracks bug reports, helping us to identity trends. Bugs can be broken down as tables with relevant properties, and session replays, attached.
+
+![bug table](../images/blog/aruba/arubug-table.png)
+
+The team also built a communication tab into bug reports, so support can send emails and leave notes on tickets with additional context without leaving the app. Every email is a ClickHouse event tied to the initial UUID of the report event, with emails (for now) sent and received via Zapier.
+
+![communication-tab](../images/blog/aruba/conversation-tab.png)
+
+After the hackathon we've reviewed the plan:
+
+- We realized that there's another problem internally. Due to the increased complexity of our product, it’s become harder for a single engineer to provide good support for all of our products. Support heroes also shield product teams from feedback about their product, which means it’s harder to iterate on a product. So we're going to use this in-app integration to split out to the more specific team helping us do triage effectively.
+
+- We've decided at this point not to continue the work on the communication tab and instead integrate better with Zendesk. That said, the way we handle support can be used by customers too – there will be a blog post coming to explain this further.
```

Comment:
```suggestion
After the hackathon, we reviewed the plan and realized: 

1. Due to the increased complexity of our product, it’s become harder for a single engineer to provide good support for all of our products. 

2. Support heroes shield product teams from important feedback about their product, making it harder iterate.

Consequently, we decided to use the hackathon in-app integration to triage support requests by product/team, and improve our integration with Zendesk instead of developing the communication tab further. 

// TODO: Add in-app bug report screenshot

Now, instead of our single support hero rotation, each team will handle support requests for their product, and tickets will be enhanced with useful context, such as session replays and product usage.
```

@tiina303 Revision of this section. Let me know if this works.

Reply:
yes this looks better :)

Note the other pro of this approach in my mind was that we now can look at aggregated info (re last sentence)>